<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateFacadeOptionDialog</class>
 <widget class="QDialog" name="GenerateFacadeOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenerateFacadeOptionDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Config</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageWidth">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>19</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageHeight">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWindowDis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window displayment</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWindowProb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>76</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window prob</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWindowDis">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWindowProb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>76</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPadding">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Padding</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPaddingMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPaddingMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDoors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ebable doors</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>595</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>595</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Facade Images</string>
   </property>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFacadeImagePath">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInputPath">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>47</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># images:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageNum">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>47</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDataAug">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Augmentation</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>292</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Window Config</string>
   </property>
   <widget class="QLineEdit" name="lineEditRelativeHeightMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNGMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>79</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNCMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NC:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelativeWidthMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNRMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NR:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNRMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNCMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Height:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelativeWidthMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NG:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelativeHeightMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>139</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNGMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>79</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>79</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>472</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Door Config</string>
   </property>
   <widget class="QLineEdit" name="lineEditDRelativeWidthMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>53</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>53</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDRelativeWidthMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>53</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>83</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDRelativeHeightMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>83</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>53</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R Width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDRelativeHeightMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>82</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNDMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNDMin">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ND:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <zorder>lineEditDRelativeWidthMax</zorder>
   <zorder>label_20</zorder>
   <zorder>lineEditDRelativeWidthMin</zorder>
   <zorder>label_6</zorder>
   <zorder>label_21</zorder>
   <zorder>lineEditDRelativeHeightMax</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEditDRelativeHeightMin</zorder>
   <zorder>lineEditNCMin_2</zorder>
   <zorder>label_23</zorder>
   <zorder>label_22</zorder>
   <zorder>lineEditNCMax_2</zorder>
   <zorder>lineEditNDMax</zorder>
   <zorder>lineEditNDMin</zorder>
   <zorder>label_22</zorder>
   <zorder>label_23</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
